src/sagemaker/inference_recommender/__init__.py

```diff
@@ -12,3 +12,29 @@
 # language governing permissions and limitations under the License.
 """Classes for using Inference Recommender with Amazon SageMaker."""
 from __future__ import absolute_import
+
+
+class Phase:
```

Comment:
would it be possible to leave the implementation in the other file and over here just do this?

```
from sagemaker.inference_recommender.inference_recommender_mixin import Phase, ModelLatencyThreshold
```